packages/cosmic-swingset/package.json

```
@@ -2,9 +2,7 @@
   "name": "@agoric/cosmic-swingset",
   "version": "0.31.10",
   "description": "Agoric's Cosmos blockchain integration",
-  "parsers": {
-    "js": "mjs"
-  },
+  "type": "module",
   "bin": {
     "ag-chain-cosmos": "./src/entrypoint.cjs"
```

This needs to change as well.  Please be aware of other appearances of `src/entrypoint.cjs` for future PRs.  I think this is the first time the move from `src/entrypoint.cjs` to `src/entrypoint.js` was made.

```suggestion
    "ag-chain-cosmos": "./src/entrypoint.js"
```